src/script/luna/mememu.cpp

```diff
@@ -1524,6 +1605,40 @@ class NPCMemory final : public NpcParent
         {
             obj->Reset[2] = value != 0;
         }
+        else if(address >= 0x0A && address < 0x14) // Pinched
+        {
+            int in;
+
+            memToValue(in, value, ftype);
+
+            // clamp to range
+            if(in < 0)
+                in = 0;
+            else if(in > 3)
+                in = 3;
+
+            switch(address)
+            {
+            case 0x0A: // Pinched1
+                obj->Pinched.Bottom1 = in;
+                return;
+            case 0x0C: // Pinched2
+                obj->Pinched.Left2 = in;
+                return;
+            case 0x0E: // Pinched3
+                obj->Pinched.Top3 = in;
+                return;
+            case 0x10: // Pinched4
+                obj->Pinched.Right4 = in;
+                return;
+            case 0x12: // MovingPinched
+                obj->Pinched.Moving = in;
+                return;
+            default:
+                // use default handler
```

Comment:
Same as above:
I guess, the warning message should be added here in any cases of attempts to hack bytes in this area.